server/controller/studentController.js

```diff
@@ -38,6 +38,7 @@ const Login = async function (req, res) {
     }
 }
 
+
```

Comment:
remove this extra space

Reply:
Line no 41 has extra space  so, i only need to remove that extra space

Reply:
Yep, this change on line 41 is not necessary. 🙂